src/App.tsx

```diff
@@ -310,27 +309,19 @@ function App() {
 
   return (
     <div className='container-fluid px-0'>
-      <nav className='navbar navbar-expand-lg navbar-light bg-light px-2'>
+      <nav className='navbar fixed-top navbar-expand-lg navbar-light bg-light px-2' style={{top:50}}>
         <span className='navbar-brand pl-4'>UCF Campus Map</span>
-        <div className='container'>
-          <button className='navbar-toggler justify-self-right' type='button' data-bs-toggle='collapse' data-bs-target='#navbarSupportedContent' aria-controls='navbarSupportedContent' aria-expanded='false' aria-label='Toggle navigation'>
-            <span className='navbar-toggler-icon'></span>
-          </button>
-          <div className='collapse navbar-collapse' id='navbarSupportedContent'>
-            <NavigationMenu
-              listClasses='navbar-nav d-flex justify-content-evenly mb-2 mb-lg-0 w-100'
-              menuId='header-menu'
-              remoteMenuId={HEADER_MENU_ID} />
-          </div>
-        </div>
-      </nav>
-      <div className='row gx-0'>
-        <div className='col-12 col-md-2 px-0 px-md-3 bg-light'>
+        <button className='navbar-toggler justify-self-right mb-1' type='button' data-bs-toggle='collapse' data-bs-target='#navbarSupportedContent' aria-controls='navbarSupportedContent' aria-expanded='false' aria-label='Toggle navigation'>
+          <span className='navbar-toggler-icon'></span>
+        </button>
+        <div className='px-0 px-md-3 bg-light navbar-collapse collapse' id="navbarSupportedContent">
           <MapMenu
             visibility={visibility}
             setVisibility={setVisibility} />
         </div>
-        <div className='col-12 col-md-10'>
+      </nav>
+      <div className='row gx-0'> 
+        <div className='col-12'>
```

Comment:
if we're always using all 12 columns, neither of these divs are needed. The `.container` div below will be sufficient.

Reply:
I tried that but there will be a design issue after that. Since there is a parent div with class of container, at lease we need to have a .row class to separate that row. 

<img width="435" alt="Screenshot 2024-06-04 at 10 35 41 AM" src="https://github.com/UCF/campus-map-react/assets/83763883/c458f891-4f05-43b5-aab5-bbe0a6523eec">

Reply:
It is very normal to use content within a container without rows or columns. Usually, if you're having an issue making that work, it's because of something in particular you've done with custom CSS - which was the case here. In order to make the layout work with the calculated min-height, we needed to make the `map-container` class a flex container with the `flex-wrap` set to wrap.